middleware.go

```diff
@@ -454,45 +431,43 @@ func handleContentType(c *Client, r *Request) {
 	}
 }
 
-func handleRequestBody(c *Client, r *Request) (err error) {
+func handleRequestBody(c *Client, r *Request) error {
 	var bodyBytes []byte
-	contentType := r.Header.Get(hdrContentTypeKey)
-	kind := kindOf(r.Body)
+	releaseBuffer(r.bodyBuf)
```

Comment:
@SVilgelm I'm wondering, why are you releasing buffer here?

Reply:
There is a good chance that the buffer has been acquired, so it is better to return it back to the pool, so it can be reused later. Anyway the bodyBuf is set to nil here

Reply:
I see.